Makefile

```diff
@@ -197,7 +197,7 @@ $(ENVTEST): $(LOCALBIN)
 .PHONY: certificates
 certificates:
 	# generate TLS certs
-	scripts/generate_certs.sh $(DOMAIN)
+	scripts/generate_certs.sh $(or $(DOMAIN),$(shell oc get ingresses.config/cluster -o jsonpath='{.spec.domain}'))
```

Comment:
shouldn't this be guarded in case both of the two fails (retrieving env variable, executing oc get) so to handle the case none are successful call somewhere instead of inline?

Reply:
Makefile didn't make it easy to add logic easily, so I added a guard in the `generate_certs.sh` script. It'll make sure that a non-empty domain is used.